features/mouse-events.yml.dist

```diff
@@ -28,6 +28,7 @@ compat_features:
   #   safari_ios: "1"
   - api.MouseEvent
   - api.MouseEvent.button
+  - api.MouseEvent.clientX
```

Comment:
This should be `api.MouseEvent.movementX` and will be fixed in 5.7.2. I don't think it's a blocker for web features.